optaplanner-docs/src/main/asciidoc/index-docinfo.html

```diff
@@ -1,3 +1,10 @@
+<!-- Populate jstree for collapsed Table of Content-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
I also noticed in the generated index.html code (target/generated-docs/html_single/index.html) that it's importing:
```css
(line 420)
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
<!-- Adobe Analytics for Red Hat - DPAL (DTM Property Auto-Loader) - part 1/2 -->
```
I'm not sure if asciidoc is generating it or somewhere in the documentation because it's not in the index-docinfo.html file